opts/color.go

```diff
@@ -0,0 +1,117 @@
+package opts
+
+import (
+	"bytes"
+	"fmt"
+	"strings"
+)
+
+type Color string
+
+type ColorVar interface {
```

Comment:
Interface for what?

If you want to use interface as the color type, I think the code shown  below will be a better approach

`Colors      []opts.Color --> []opts.ColorVar`